apps/courses/src/routes/index.tsx

```diff
@@ -24,54 +29,192 @@ const UnauthRoute = (props) => {
+      <AuthRoute
+        path="settings"
+        element={
+          <RouteWrapper>
+            <Settings />
+          </RouteWrapper>
+        }
+      />
+      <Route
+        path=":uid"
+        element={
+          <RouteWrapper>
+            <Profile />
+          </RouteWrapper>
+        }
+      />
     </Route>
     <Route path="courses">
-      <Route path="/" element={<CoursePage which="search" />} />
+      <Route
+        path="/"
+        element={
+          <RouteWrapper>
+            <CoursePage which="search" />
+          </RouteWrapper>
+        }
+      />
       <Route path=":course">
-        <Route path="/" element={<CoursePage which="overview" />} />
+        <Route
+          path="/"
+          element={
+            <RouteWrapper>
+              <CoursePage which="overview" />
+            </RouteWrapper>
+          }
+        />
         <AuthRoute
           path="complete"
-          element={<CoursePage which="courseComplete" />}
+          element={
+            <RouteWrapper blackHeader>
+              <CoursePage which="courseComplete" />
+            </RouteWrapper>
+          }
         />
         <AuthRoute path="project">
-          <AuthRoute path="/" element={<CoursePage which="project" />} />
+          <AuthRoute
+            path="/"
+            element={
+              <RouteWrapper noHeader>
+                <CoursePage which="project" />
+              </RouteWrapper>
+            }
+          />
           <AuthRoute
             path="complete"
-            element={<CoursePage which="projectComplete" />}
+            element={
+              <RouteWrapper blackHeader noFooter>
+                <CoursePage which="projectComplete" />
+              </RouteWrapper>
+            }
           />
           <AuthRoute path=":part">
             <AuthRoute
               path="/"
-              element={<CoursePage which="projectSubmission" />}
+              element={
+                <RouteWrapper noHeader>
+                  <CoursePage which="projectSubmission" />
+                </RouteWrapper>
+              }
             />
             <AuthRoute
               path=":attempt"
-              element={<CoursePage which="projectSubmission" />}
+              element={
+                <RouteWrapper noHeader>
+                  <CoursePage which="projectSubmission" />
+                </RouteWrapper>
+              }
             />
           </AuthRoute>
         </AuthRoute>
         <AuthRoute path=":lesson">
-          <AuthRoute path="/" element={<CoursePage which="lesson" />} />
+          <AuthRoute
+            path="/"
+            element={
+              <RouteWrapper noHeader>
+                <CoursePage which="lesson" />
+              </RouteWrapper>
+            }
+          />
           <AuthRoute
             path="complete"
-            element={<CoursePage which="lessonComplete" />}
+            element={
+              <RouteWrapper blackHeader inConcept noFooter>
```

Comment:
There should be `noHeader` here, and not `blackHeader`. The black header you're seeing in the designs is the course `<Header>` component, which is always black. The only page that has the main `<Header>` component as black is the course complete page.

Also, no need for `inConcept`, see first comment.